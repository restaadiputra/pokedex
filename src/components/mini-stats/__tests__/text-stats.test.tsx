import React from 'react';
import { render, screen } from '@testing-library/react';
import TextStats from '../text-stats';

test('should render the field and its value', () => {
  const { container, rerender } = render(
    <TextStats field="Height" value="100m" />
  );
  expect(screen.getByText('Height :')).toBeInTheDocument();
  expect(screen.getByText('100m')).toBeInTheDocument();
  expect(container).toMatchSnapshot();

  rerender(<TextStats field="Width" value="100kg" />);
  expect(screen.getByText('Width :')).toBeInTheDocument();
  expect(screen.getByText('100kg')).toBeInTheDocument();
});
