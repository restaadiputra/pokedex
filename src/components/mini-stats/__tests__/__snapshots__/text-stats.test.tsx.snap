// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the field and its value 1`] = `
<div>
  <p
    class="text-white tracking-wide text-sm opacity-75 font-light"
  >
    <span
      class="font-normal"
    >
      Height : 
    </span>
    100m
  </p>
</div>
`;
